store/actions/threads.js

```diff
@@ -0,0 +1,110 @@
+// Module imports
+import 'isomorphic-fetch'
```

Comment:
We don't actually need `isomorphic-fetch` anymore, `axios` takes care of that for us.

Reply:
Fixed. whoops.